src/main/steps/tasklistresponse/utils.ts

```diff
@@ -55,3 +71,13 @@ export const getPartyDetails = (userCase: CaseWithId, userId: UserDetails['id'])
 
   return partyData?.value ? partyData.value : partyData;
 };
+
+export const getRespondentDetails = (userCase: CaseWithId, userId: UserDetails['id']): Respondent => {
```

Comment:
Remove this unnecessary method instead change the mappers to accept partydetails

Reply:
Yes Removed